requirements-common.txt

```diff
@@ -4,6 +4,7 @@ regex
 pandas
 kagglehub
 keras-core>=0.1.6
+keras_nlp
```

Comment:
We don't want to add to `reqirements_common`. This will cause other issues since `keras_nlp` will install TF with it and we may want the GPU version. So lets keep `keras_nlp` out of requirements file.